### PR TITLE
VIH-7878 - Update to wait for web element to disappear

### PR DESCRIPTION
**Pull request by MichaelKeaneathmcts**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

**Review by robertbennett1998 — APPROVED** [review]
LGTM

File: AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs [hunk 1]
```diff
@@ -183,7 +184,12 @@ public void Click(By element, int timeout = 20)
             wait.Until(drv => Driver.FindElement(element)).Click();
         }
 
-
+        public void WaitForElementToNotExist(By element, int timout = 300)
```
Comment (FarronF) [inline AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs:187]: timout should be timeout

Comment (FarronF) [reply AcceptanceTests/AcceptanceTests.Common/Driver/Drivers/UserBrowser.cs:187]: Also it isn't being used in this method.

File: AcceptanceTests/AcceptanceTests.Common/Driver/Helpers/WaitHelper.cs [hunk 2]
```diff
@@ -12,7 +12,11 @@ class WaitHelper
     {
         public WebDriverWait newWait(NgWebDriver Driver)
         {
-            WebDriverWait wait = new WebDriverWait(Driver, timeout: TimeSpan.FromSeconds(30))
+            return newWait(Driver, 30);
+        }
```
Comment (FarronF) [inline AcceptanceTests/AcceptanceTests.Common/Driver/Helpers/WaitHelper.cs:16]: If this method is just going to call the other with 30 could we just make that an optional argument instead?

**Pull request merged**